modules/n1ql/pages/n1ql-language-reference/searchfun.adoc

```diff
@@ -118,10 +118,10 @@ The object may contain the following fields:
 | `index`
 [Optional]
 | string, object
-| The `index` field may be a string, containing the name of a full text index in the keyspace.
-(This may be a full text index alias, but only if the full text index is in the same keyspace.)
+| The `index` field may be a string, containing the name of a Search index in the keyspace.
+(This may be a Search index alias, but only if the Search index is in the same keyspace.)
```

Comment:
I'd suggest using "can/might" instead of "may" in docs. "May" can be ambiguous, especially for ESL speakers.

Reply:
I didn't write most of this, but I can change this while I'm in here. I was just tweaking some existing names/phrases.